aspnetcore/blazor/components/index.md

```diff
@@ -783,6 +783,96 @@ In the preceding example:
 
+:::moniker-end
+
+:::moniker range=">= aspnetcore-8.0"
+
+### Dispatch exceptions to Blazor's synchronization context
+
+Use `ComponentBase.DispatchExceptionAsync` in Razor components to process exceptions thrown outside of Blazor's synchronization context. This permits the app's code to treat the exceptions as through they're lifecycle method exceptions. Thereafter, Blazor's error handling mechanisms, such as [error boundaries](xref:blazor/fundamentals/handle-errors#error-boundaries), can process the exceptions.
+
+> [!NOTE]
+> `ComponentBase.DispatchExceptionAsync` is used in Razor component files (`.razor`) that inherit from <xref:Microsoft.AspNetCore.Components.ComponentBase>. When creating components that [implement <xref:Microsoft.AspNetCore.Components.IComponent> directly](#component-classes), use `RenderHandle.DispatchExceptionAsync`.
+
+To dispatch exceptions to Blazor's synchronization context, use a `try-catch` block to pass the exception to `DispatchExceptionAsync` and await the result:
+
+```csharp
+try
+{
+    await InvokeAsync(() =>
+    {
+        ...
+    });
+}
+catch (Exception ex)
+{
+    await DispatchExceptionAsync(ex);
+}
+```
+
+For an example use of `DispatchExceptionAsync` API, implement the timer notification example in the [Invoke component methods externally to update state](#invoke-component-methods-externally-to-update-state) section. The example uses a timer outside of Blazor's synchronization context, where an unhandled exception normally doesn't reach Blazor's synchronization context and isn't processed by Blazor's error handling mechanisms, such as an [error boundary](xref:blazor/fundamentals/handle-errors#error-boundaries).
+
+First, change the code in `TimerService.cs` to create an artificial exception outside of Blazor's synchronization context. In the `while` loop of `TimerService.cs`, throw an exception when the `elapsedCount` reaches a value of two:
+
+```csharp
+if (elapsedCount == 2)
+{
+    throw new Exception("I threw an exception! Somebody help me!");
+}
+```
+
+Place an [error boundary](xref:blazor/fundamentals/handle-errors#error-boundaries) in the app's main layout.
+
+`Shared/MainLayout.razor`:
+
+```razor
+<article class="content px-4">
+    <ErrorBoundary>
+        <ChildContent>
+            @Body
+        </ChildContent>
+        <ErrorContent>
+            <p class="alert alert-danger" role="alert">
+                Oh, dear! Oh, my! - George Takei
+            </p>
+        </ErrorContent>
+    </ErrorBoundary>
+</article>
+```
+
+Normally, the preceding error boundary doesn't react to unhandled exceptions thrown outside of Blazor's synchronization context. If you run the app at this point, the exception is thrown when the elapsed count reaches a value of two, but the error boundary doesn't show the error content.
```

Comment:
The `Normally` implies, that there is a "trick" to do and it will show, but given the current state of code, it won't. Hence, I think we should remove the `Normally,` part.

```suggestion
If you run the app at this point, the exception is thrown when the elapsed count reaches a value of two, but nothing on the UI changes and the error boundary doesn't show the error content.
```

Reply:
Sounds good, but we're running on a bit with a looooong sentence. Let's break it it up into separate thoughts (probably better for MT anyway) .....

```suggestion
If you run the app at this point, the exception is thrown when the elapsed count reaches a value of two. However, nothing in the UI changes. The error boundary doesn't show the error content.
```